capstone/src/main/webapp/index.css

```diff
@@ -0,0 +1,114 @@
+html {
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  background-color: black;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100vh;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: sans-serif;
+}
+
+.titlecontainer {
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+}
+
+.titlecontainer span {
+  text-transform: uppercase;
+  display: block;
```

Comment:
I think alphabetizing the definitions within the block is best practice according to [https://google.github.io/styleguide/htmlcssguide.html](style guide)

Reply:
true, odd it passed linter lol